tests/utils.py

```diff
@@ -143,3 +145,19 @@ def run_pattern(pattern, data, metadata):
     reverse_transformed = pattern.reverse_transform(transformed)
 
     return updated_metadata, transformed, reverse_transformed
+
+
+def run_copula(data, metadata, pattern):
+    synthesizer = GaussianCopulaSynthesizer(metadata)
+    synthesizer.add_cag(patterns=pattern)
+    synthesizer.fit(data)
+
+    return synthesizer
+
+
+def run_hma(data, metadata, pattern):
+    synthesizer = HMASynthesizer(metadata)
+    synthesizer.add_cag(patterns=[pattern])
```

Comment:
minor: can we have both of these test helpers take the same type of argument for `pattern` for consistency? Also can we rename it to `patterns` or `pattern_list` if it remains a list type?